core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcAdminImportTestUtils.java

```diff
@@ -723,6 +810,24 @@ private List<TestData> createExistingSqlServerDatabaseWithAllDataTypes(String na
     return ImmutableList.copyOf(data);
   }
 
+  private ImmutableList<String> getIntCompatibleColumnNamesOnExistingSqlServerDatabase(
+      String table) {
+    if (table.equals(SUPPORTED_TABLE_NAME)) {
+      return ImmutableList.of("col02", "col03", "col04");
```

Comment:
[random thought] For future reference: adding type hints to these column names (e.g., "col02_int") might reduce confusion. Not related to this PR, just something I noticed.